### PR TITLE
Adding CartoonGAN model

### PR DESCRIPTION
**Pull request by sayakpaul**
This PR contains the CartoonGAN model as proposed [here](https://github.com/SystemErrorWang/White-box-Cartoonization/blob/master/paper/06791.pdf). Some sample results:

image

image

Cc: @margaretmz @khanhlvg

### CODE REVIEW

File: tfhub_dev/assets/sayakpaul/cartoongan/dr/1.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Lite sayakpaul/east-text-detector/dr/1
```
Comment (maringeo) [inline tfhub_dev/assets/sayakpaul/cartoongan/dr/1.md:1]: This handle is already used in https://raw.githubusercontent.com/tensorflow/hub/master/tfhub_dev/assets/sayakpaul/east-text-detector/dr/1.md

Is it possible to choose a different handle?

Comment (sayakpaul) [reply tfhub_dev/assets/sayakpaul/cartoongan/dr/1.md]: Fixed it up.

**Comment by sayakpaul** [comment]
@maringeo any updates on this?

**Comment by maringeo** [comment]
> @maringeo any updates on this?

Hi @sayakpaul, the placeholder model is published but it looks like the Lite models were not processed correctly: https://tfhub.dev/sayakpaul/cartoongan/1. This is an internal bug and I'm looking into it. I'll update this thread once I figure out the problem.

**Comment by sayakpaul** [comment]
> > @maringeo any updates on this?
> 
> Hi @sayakpaul, the placeholder model is published but it looks like the Lite models were not processed correctly: https://tfhub.dev/sayakpaul/cartoongan/1. This is an internal bug and I'm looking into it. I'll update this thread once I figure out the problem.

Thanks for letting me know, @maringeo.

**Comment by maringeo** [comment]
Hi @sayakpaul, thank you for contributing to TF Hub! The models are live on https://tfhub.dev/sayakpaul/lite-model/cartoongan/dr/1.

This PR has been merged in 7f320e0eb8b167aef06a1753eed9e04f0cf5b519

**Comment by sayakpaul** [comment]
Thanks, @maringeo for letting me know.

**Pull request closed without merging**